README.md

```diff
@@ -95,8 +81,55 @@ When calling GET /am?prefix={complete-permission-here} your server should respon
 
 When an unauthorized request is made, a response with `{ "permission": {string}, "alias": {string} }` is expected.
 
+### The CI/CD pipeline
+
+Will.IAM has a very simple CI/CD pipeline in place to help us guarantee that the code have a good quality and avoid broken releases.
+Currently we use TravisCI to automate the execution of tests, code quality tools and generation and publishing of images in our 
+[Docker Hub repository](https://hub.docker.com/r/tfgco/will-iam). The Pipeline works as follows:
+
+![](ci_pipeline.jpg)
+
+### Issuing new releases
+
+Versioning happens through the `version.txt` file, which stores the project's current version.
+Given the project's current situation, the current workflow expects that releases will
+be launched often, containing small increments, hence the tight integration with Pull Requests. In order to
```

Comment:
nit: I don't get it, are you line breaking at a certain column # or not? Please break consistently (or not at all).